uptane-standard.md

```diff
@@ -209,7 +215,9 @@ These terms are defined in greater detail in {{roles}}.
 
 *ECUs*: Electronic Control Units, the computing units on a vehicle
 
-*LIN Bus*: Local Interconnect Bus  
+*LIN Bus*: Local Interconnect Bus 
+
+*RXSWIN*: RX Software Identification Number, defined in UN WP29 as "a dedicated identifier, defined by the vehicle manufacturer, representing information about the type approval relevant software of the Electronic Control System contributing to the Regulation N*X type approval relevant characteristics of the vehicle." 
```

Comment:
Why do we want to specify this if it is completely optional?

Reply:
Agreed

Reply:
I believe it is helpful for implementer's to understand where international standards (UN WP 29 - utilizing RXSWIN) can be included in the metadata.

Please let me know if you do not believe this is helpful.

Reply:
I will remove, with proposal of including in Implementation Guide/Considerations.